app/assets/javascripts/components/course_creator/course_creator.jsx

```diff
@@ -192,18 +211,25 @@ const CourseCreator = createReactClass({
     if (this.props.loadingUserCourses) {
       return <div />;
     }
-    // There are three fundamental states: NewOrClone, CourseForm, and CloneChooser
+    // There are four fundamental states: NewOrClone, CourseForm, wizardForm and CloneChooser
     let showCourseForm;
     let showCloneChooser;
     let showNewOrClone;
+    let showWizardForm;
     // If user has no courses, just open the CourseForm immediately because there are no cloneable courses.
     if (this.props.cloneableCourses.length === 0) {
+      if (this.state.showWizardForm) {
+        showWizardForm = true;
+      }
```

Comment:
Since these states are mutually exclusive, this should probably be an if/else, rather than just the if.

Reply:
okay